### ISSUE TITLE
old issue "Right of current tab" disfunction reverted

### ISSUE THREAD

**Issue by bravo-hero** [jaden/Tab-Position-Customizer-2#5]
Hi Team,

regarding the #3 , 
it feels better, but, it still got the similar issue. not 1 time, not 2 times, but several times already.
it's kind of the process:

long time no use the chrome window, then tabs-> got freezing(i guess)->at one time i come back to create a new tab->
IT CREATED IN THE RIGHTEST OF THE CHROME AS THE DEFAULT BEHAVIOR->i noticed the issue, i create a new tab one more time->it created in the right as the current active tab(as expected)->this issue urge me to come back to report this issue back.

thanks.

**Comment by farazoman** [comment]
it looks like the issue from Chrome has been fixed but not available yet on the official build (according to the latest chrome version I'm using). 

https://bugs.chromium.org/p/chromium/issues/detail?id=1213925
My current Chrome version: Version 91.0.4472.114 (Official Build) (x86_64)
It should be available on this build of chrome: Chrome 91.0.4472.119

The fix I pushed was just a quick fix to workaround the issue. I noticed after using it for a bit that it wasn't perfect either but works better than before.
